milksnake/setuptools_ext.py

```diff
@@ -141,9 +141,13 @@ def __init__(self, spec, cmd=None, cwd=None, path=None, env=None):
         self.env = env
 
     def find_dylib(self, name, in_path=None):
-        path = self.path or '.'
+        path = self.path or os.curdir
         if in_path is not None:
-            path = os.path.join(path, *in_path.split('/'))
```

Comment:
@mitsuhiko why do we do this in the first place? What speaks against `os.path.join(path, in_path)`?